src/pages/commits/commitChart/ChartToggle.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { RadioGroup, Radio } from "@leafygreen-ui/radio-group";
+import { Label } from "@leafygreen-ui/typography";
+import { useLocation, useHistory } from "react-router-dom";
+import { ChartToggleQueryParams, ChartTypes } from "types/commits";
+import { queryString } from "utils";
+
+const { gray } = uiColors;
+const { stringifyQuery } = queryString;
+
+export const ChartToggle: React.FC<{
+  currentChartType: ChartTypes;
+}> = ({ currentChartType }) => {
+  const { pathname } = useLocation();
+  const { replace } = useHistory();
+
+  const onChangeChartType = (
+    event: React.ChangeEvent<HTMLInputElement>
+  ): void => {
+    const nextChartType = event.target.value as ChartTypes;
+    replace(
+      `${pathname}?${stringifyQuery({
+        [ChartToggleQueryParams.chartType]: nextChartType,
+      })}`
+    );
+  };
+
+  return (
+    <Container>
+      <ToggleWrapper>
+        <Label htmlFor="chart-toggle">View Options</Label>
+        <StyledRadioGroup
+          size="default"
+          onChange={onChangeChartType}
+          value={currentChartType}
+          name="chart-select"
+        >
+          <Radio
+            data-cy="chart-absolute-radio"
+            id="cy-chart-absolute-radio"
+            value={ChartTypes.Absolute}
+          >
+            <Label htmlFor="chart-radio-absolute">Absolute Number</Label>
```

Comment:
`htmlFor` is an accessibility-related prop used to identify which input element a label is associated with. This means that the contents of `htmlFor` should match the input's `id`, so here, the ID on line 42 should be `chart-radio-absolute`.